src/status_im2/contexts/quo_preview/navigation/page_nav.cljs

```diff
@@ -2,11 +2,17 @@
   (:require [clojure.string :as string]
             [quo2.core :as quo]
             [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as quo.resources]
             [react-native.core :as rn]
             [reagent.core :as reagent]
             [status-im2.common.resources :as resources]
             [status-im2.contexts.quo-preview.preview :as preview]))
 
+(def ^:private networks
```

Comment:
I think we have this defined in multiple places now? Should we put it somewhere common(if so) ? 🤔

Reply:
Agree, maybe in `status-im2.config` namespace?

Reply:
Yes, We should have the supported (L2) chains in the config and from the config we need to generate the image source array.